src/components/mediaLibraryEditor/mediaLibraryEditor.js

```diff
@@ -31,6 +31,14 @@ function onEditLibrary() {
     loading.show();
     const dlg = dom.parentWithClass(this, 'dlg-libraryeditor');
     let libraryOptions = libraryoptionseditor.getLibraryOptions(dlg.querySelector('.libraryOptions'));
+    // when the library has moved or symlinked, the ItemId is not correct anymore
+    // this can lead to a forever spinning value on edit the library parameters
+    if (currentOptions.library.ItemId == undefined) {
+        toast('The library setting is in an invalid state, cannot edit. You are most likely suffering from a bug where the path in the database is not the absolute path in the filesystem');
```

Comment:
1. Make this text translatable: move this text to `strings/en-us.json` and use `globalize.translate`.
2. I think we need to use an alert so that the user has enough time to read. Similar to this:
https://github.com/jellyfin/jellyfin-web/blob/1a9f51c9b0d589f81e09c0614dbe360ee8e78307/src/plugins/experimentalWarnings/plugin.js#L32
You will probably have to close this dialog first and then open alert.